src/dns_server.c

```diff
@@ -1306,9 +1306,20 @@ static int _dns_server_setup_ipset_packet(struct dns_server_post_context *contex
 				}
 
 				/* add IPV4 to ipset */
-				ipset_add(rule->ipsetname, addr, DNS_RR_A_LEN, request->ip_ttl * 2);
-				tlog(TLOG_DEBUG, "IPSET-MATCH: domain: %s, ipset: %s, IP: %d.%d.%d.%d", request->domain,
-					 rule->ipsetname, addr[0], addr[1], addr[2], addr[3]);
+#ifdef WITH_NFTSET
```

Comment:
we can support both ipset and nftset at same time.

Reply:
make sense. my thought was to follow some openwrt packages, e.g. miniupnd, whcih only support one backend: user can choose to install miniupnpd-iptables or miniupnpd-nftables. but anyway it make senses to support  both backends.